internal/indexer/transactions.go

```diff
@@ -27,9 +28,12 @@ func (bc *blockContext) processTransactions(trxHashes *[]string) (err error) {
 				}
 				internal := makeInternal(trx_model, entry)
 				internal_transactions.Data = append(internal_transactions.Data, internal)
+				eventLog := makeEventLog(bc, trx, entry)
+				logs.Data = eventLog
```

Comment:
1. This shouldn't be in this trace for cycle, but after it 
2. Looks like you don't need `makeEventLog` at all as you are just copying the same data from one place to another. So logs.Data = trx.Logs should work just fine.